requirements.txt

```diff
@@ -19,3 +19,6 @@ docutils   # pinning this because it breaks at 0.18
 
 # for packaging
 wheel
+
+# for polishing up the Pip install
+PLAN-Tools==0.5
```

Comment:
PLAN-Tools is now a dependency for both building out the configuration entry point as well as fixing up the taskbar icon on Windows.